apps/angular/src/app/features/anime-page/anime-page.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component } from '@angular/core';
+import { AnimePagination } from '@js-camp/core/models/anime';
+import { BehaviorSubject, Observable, debounceTime, shareReplay, switchMap, tap } from 'rxjs';
+import { PageEvent } from '@angular/material/paginator';
+import { DEBOUNCE_TIME } from '@js-camp/angular/core/utils/constants';
+
+import { AnimeService } from '../../../core/services/anime.service';
+
+/** Anime Component. */
+@Component({
+	selector: 'camp-anime-page',
+	templateUrl: './anime-page.component.html',
+	styleUrls: ['./anime-page.component.css'],
+})
+export class AnimePageComponent {
+	/** Status of anime getting from server. */
+	public isLoading$ = new BehaviorSubject<boolean>(false);
+
+	/** Current page index. */
+	public page$ = new BehaviorSubject<number>(0);
+
+	/** Anime response. */
+	public animeResponse$ = new Observable<AnimePagination>();
```

Comment:
Why you declare instance? Please add only type
```suggestion
	public readonly animeResponse$: Observable<AnimePagination>;
```

And please not forget about readonly